yarn.lock

```diff
@@ -8263,7 +8263,7 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react-dom@>=16.9.0":
+"@types/react-dom@>=16.9.0", "@types/react-dom@^17.0.9":
```

Comment:
Non-blocking, but why did this happen? 🤔 The resolved version is the same and all.

Reply:
looks like it's changed before and now it was fixed 🤔